public/assets/jquery-ui/effect-puff.js

```diff
@@ -28,7 +28,7 @@ return t.effects={effect:{}},function(t,e){function n(t,e,n){var r=l[e.type]||{}
ze=function(e,i){var n,o,r,s=t(this),a=["position","top","bottom","left","right","width","height","overflow","opacity"],l=["position","top","bottom","left","right","overflow","opacity"],u=["width","height","overflow"],c=["fontSize"],h=["borderTopWidth","borderBottomWidth","paddingTop","paddingBottom"],p=["borderLeftWidth","borderRightWidth","paddingLeft","paddingRight"],d=t.effects.setMode(s,e.mode||"effect"),f=e.restore||"effect"!==d,m=e.scale||"both",g=e.origin||["middle","center"],y=s.css("position"),v=f?a:l,b={height:0,width:0,outerHeight:0,outerWidth:0};"show"===d&&s.show(),n={height:s.height(),width:s.width(),outerHeight:s.outerHeight(),outerWidth:s.outerWidth()},"toggle"===e.mode&&"show"===d?(s.from=e.to||b,s.to=e.from||n):(s.from=e.from||("show"===d?b:n),s.to=e.to||("hide"===d?b:n)),r={from:{y:s.from.height/n.height,x:s.from.width/n.width},to:{y:s.to.height/n.height,x:s.to.width/n.width}},("box"===m||"both"===m)&&(r.from.y!==r.to.y&&(v=v.concat(h),s.from=t.effects.setTransition(s,h,r.from.y,s.from),s.to=t.effects.setTransition(s,h,r.to.y,s.to)),r.from.x!==r.to.x&&(v=v.concat(p),s.from=t.effects.setTransition(s,p,r.from.x,s.from),s.to=t.effects.setTransition(s,p,r.to.x,s.to))),("content"===m||"both"===m)&&r.from.y!==r.to.y&&(v=v.concat(c).concat(u),s.from=t.effects.setTransition(s,c,r.from.y,s.from),s.to=t.effects.setTransition(s,c,r.to.y,s.to)),t.effects.save(s,v),s.show(),t.effects.createWrapper(s),s.css("overflow","hidden").css(s.from),g&&(o=t.effects.getBaseline(g,n),s.from.top=(n.outerHeight-s.outerHeight())*o.y,s.from.left=(n.outerWidth-s.outerWidth())*o.x,s.to.top=(n.outerHeight-s.to.outerHeight)*o.y,s.to.left=(n.outerWidth-s.to.outerWidth)*o.x),s.css(s.from),("content"===m||"both"===m)&&(h=h.concat(["marginTop","marginBottom"]).concat(c),p=p.concat(["marginLeft","marginRight"]),u=a.concat(h).concat(p),s.find("*[width]").each(function(){var i=t(this),n={height:i.height(),width:i.width(),outerHeight:i.outerHeight(),outerWidth:i.outerWidth()};f&&t.effects.save(i,u),i.from={height:n.height*r.from.y,width:n.width*r.from.x,outerHeight:n.outerHeight*r.from.y,outerWidth:n.outerWidth*r.from.x},i.to={height:n.height*r.to.y,width:n.width*r.to.x,outerHeight:n.height*r.to.y,outerWidth:n.width*r.to.x},r.from.y!==r.to.y&&(i.from=t.effects.setTransition(i,h,r.from.y,i.from),i.to=t.effects.setTransition(i,h,r.to.y,i.to)),r.from.x!==r.to.x&&(i.from=t.effects.setTransition(i,p,r.from.x,i.from),i.to=t.effects.setTransition(i,p,r.to.x,i.to)),i.css(i.from),i.animate(i.to,e.duration,e.easing,function(){f&&t.effects.restore(i,u)})})),s.animate(s.to,{queue:!1,duration:e.duration,easing:e.easing,complete:function(){0===s.to.opacity&&s.css("opacity",s.from.opacity),"hide"===d&&s.hide(),t.effects.restore(s,v),f||("static"===y?s.css({position:"relative",top:s.to.top,left:s.to.left}):t.each(["top","left"],function(t,e){s.css(e,function(e,i){var n=parseInt(i,10),o=t?s.to.left:s.to.top;return"auto"===i?o+"px":n+o+"px"})})),t.effects.removeWrapper(s),i()}})}}),/*!
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Did you mean to return a conditional instead of an assignment?

<!-- Comment by [SideCI](https://sideci.com) -->